### ISSUE TITLE
[Bug]: Pages redirected to are not rendered

### ISSUE THREAD

**Issue by thombruce** [thombruce/tnt-2024#108]
### What happened?

We have a page in TNT Docs, `/docs/getting-started/installation`, which presently yields a "Page not found" in the live environment.

Reason I believe this is the case:

The link to the page just isn't rendered anywhere. It _ought_ to be rendered on the page which ultimately redirects to it, but since this page redirects... the link isn't actually rendered.

A naive solution might be to add all pages to the hidden links in the footer... naive because this creates an enormous number of hidden elements to be rendered, especially for like a blog with thousands of posts.

We ideally want a means by which we can guarantee the link is being generated/crawled only as needed to avoid having to render countless numbers of these links somewhere they aren't...

Note, the documentation for `navigateTo` shows it being used with an `await`: https://nuxt.com/docs/api/utils/navigate-to I'm not currently doing this; perhaps it would solve the problem since... we'd be getting the resolution of the Promise (a page) rather than the unresolved Promise - this could be the issue.

### Module

TNT Content

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by thombruce** [comment]
Note the complete generated code for the redirect page (using redirect.vue template):

```html
<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=/docs/getting-started/installation"></head></html>
```

So the `<template>` tag is completely ignored, no links are rendered at all. The page just redirects...

But it's doing automatically something which I used to do manually anyway, and if I revert to the manual approach (which would use exactly the same principle to redirect) I could restore the `<template>` part and render the links... The entire contents could even be hidden if we preferred.

That seems like a fair resolution to the problem to me.

**Comment by thombruce** [comment]
Here's the original change we essentially need to revert: https://github.com/thombruce/tnt/commit/4c1f127838829c3262e84055dedc7831c6cde5b6

**Comment by thombruce** [comment]
Appears to be fixed by d2a0a182504fe60784073b5148ffc363a87d398a
